app/controllers/api/feedbacks_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::FeedbacksController < Api::BaseController
+
+  skip_before_filter :authenticate_user
```

Comment:
We want to call the helper filter `authenticate_user` in this controller since there should be a user that makes a feedback (this might just be a copy past mistake).